### PR TITLE
Display MB instead of bytes when downloading base firmware

### PR DESCRIPTION
**Pull request by EfogDev**
Because cmon, bytes :)

### CODE REVIEW

**Review by balika011 — CHANGES_REQUESTED** [review]

File: installer/install.py [hunk 1]
```diff
@@ -270,7 +270,9 @@ def traverse_ino(path, ino):
                         break
                     f.write(buf)
                     curlen += len(buf)
-                    report_interim_progress(f"Downloading... ({curlen} / {totlen} bytes)")
+                    curlen_mib = round(int(curlen) / 1024 / 1024, 2)
+                    totlen_mib = round(int(totlen) / 1024 / 1024, 2)
+                    report_interim_progress(f"Downloading... ({curlen_mib} / {totlen_mib} MiB)")
```
[the remark this answers is not in the document]
Comment (EfogDev) [reply installer/install.py]: @balika011 ok its up to you however men in black would surely be cooler

**Comment by jwise** [comment]
The reason this is taking forever to get merged is that none of us actually ever run `kexec_ui` all that often.  I didn't test it but I'll take the blame if it doesn't work.  r+ me.

**Comment by jwise** [comment]
ramboing this in despite CI failing because CI necessarily fails on forks because of the annoying key crypto thing

**Comment by singingtelegram** [comment]
Why was it changed from MiB to MB? In a Linux environment MiB is more common and if MB is used it should be multiples of 1000

**Pull request merged**